### PR TITLE
fix(experiments): add exposure column when variants > 4

### PR DESCRIPTION
**Pull request by jurajmajerik**
## Changes
Add the _exposure_ column for when the variants count > 4

image

## How did you test this code?
👀

### CODE REVIEW

**Review by neilkakkar — APPROVED** [review]
very nice :)

Kinda sus no snapshots update, but I assume this is because we're missing snapshots for > 4 variants case, bonus points if you add it 🙈

File: frontend/src/scenes/experiments/ExperimentResult.tsx [hunk 1]
```diff
@@ -67,6 +67,14 @@ export function ExperimentResult(): JSX.Element {
                                 </Col>
                             ))}
                         </div>
+                        <div className="flex justify-between py-2 border-t">
```
Comment (neilkakkar) [inline frontend/src/scenes/experiments/ExperimentResult.tsx:70]: small nit for Line 60: Can you update it to say metric when appropriate as well, like here: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/experiments/ExperimentResult.tsx#L107

Comment (jurajmajerik) [reply frontend/src/scenes/experiments/ExperimentResult.tsx:70]: eh, I'm not sure I understand, can you double check? 👇 (assuming you meant line 70)

Comment (neilkakkar) [reply frontend/src/scenes/experiments/ExperimentResult.tsx:70]: Sorry for not being clear, I meant line 60. Just calling it Exposure here is fine, but when using experiment goals like 'avg per user', calling the number as "count" doesn't make sense. We do this right for < 4 variants, but it still says "Count" always for > 4 variants

File: frontend/__snapshots__/scenes-app-experiments--running-trend-experiment-many-variants--dark.png
Comment (neilkakkar) [inline frontend/__snapshots__/scenes-app-experiments--running-trend-experiment-many-variants--dark.png:1]: @Twixes do you know why the snapshot wouldn't show the full page / is there an option to snapshot the whole page?

Comment (Twixes) [reply frontend/__snapshots__/scenes-app-experiments--running-trend-experiment-many-variants--dark.png:1]: Hm, should show the whole page. Will look into it

**Pull request merged**